src/App.js

```diff
@@ -1,5 +1,22 @@
+import { Console } from '@woowacourse/mission-utils';
+import CalculatorController from './controller/CalculatorController.js';
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      let result;
+      const input = await Console.readLineAsync(
+        '덧셈할 문자열을 입력해 주세요.\n',
+      );
+
+      result = await CalculatorController.processInput(input);
+
+      Console.print(`결과 : ${result}`);
```

Comment:
```suggestion
      const input = await Console.readLineAsync(
        '덧셈할 문자열을 입력해 주세요.\n',
      );

      const result = await CalculatorController.processInput(input);

      Console.print(`결과 : ${result}`);
```

`let` 키워드를 사용해서 result를 저장하는 이유가 있으실까요?

따로 변경이 필요한 곳에 사용되는게 아니라면 `const` 키워드를 사용하면 좋을 거 같아요

만약 `let` 키워드를 사용해야 한다면 try문의 최상단이 아니라 result를 결과를 반환하는 곳과 가까이 두는게 좋아보여요

Reply:
Class 구현순서( 상수->멤버 변수->생성자->메서드)를 지키려고 `let`을 사용했었는데,
제가 잘못 이해하고 있었던 것 같아요
코드까지 적어주셔서 감사합니다 ㅎㅎ!